src/app/store/rubrics.reducer.spec.ts

```diff
@@ -0,0 +1,258 @@
+import { DisplayableErrorInterface } from '../object-interfaces/displayable-error.interface';
+import { RubricInterface } from '../object-interfaces/rubric.interface';
+import {
+    ConnectionLost,
+    ConnectionMade,
+    CreateRubric,
+    CreateRubricError,
+    CreateRubricSuccess,
+    DeleteRubric,
+    DeleteRubricError,
+    DeleteRubricSuccess,
+    GetRubric,
+    GetRubricError,
+    GetRubrics,
+    GetRubricsError,
+    GetRubricsSuccess,
+    GetRubricSuccess,
+    SetPageTitle,
+    UpdateCurrentRubric,
+} from './rubrics.actions';
+import { rubricsReducer } from './rubrics.reducer';
+import { getInitialState } from './rubrics.state';
+
+describe('RubricsReducer', () => {
+    const { rubrics: initialState } = getInitialState();
+    const rubric: RubricInterface = { name: 'test' };
+    const currentRubric: RubricInterface = { maxLevelsCount: 0, name: 'test', totalPoints: 0 };
+    const allRubrics: Array<RubricInterface> = [];
+    const error: DisplayableErrorInterface = { message: 'test' };
+
+    describe('Undefined action', () => {
+        it('should return the default state', () => {
+            const action = { type: 'undefined' } as any;
+
+            const result = rubricsReducer(initialState, action);
+
+            void expect(result).toBe(initialState);
```

Comment:
It looks like you figured out the difference between `toBe` and `toEqual`. Nice! This took me a while and I mixed it up for a long time which led to all sorts of intermittent errors, haha.